client/src/main/java/com/influxdb/client/internal/BucketsApiImpl.java

```diff
@@ -126,6 +127,18 @@ public Buckets findBuckets(@Nonnull final FindOptions findOptions) {
 
     }
 
+    @Nonnull
+    @Override
+    public List<Bucket> findBuckets(BucketsQuery query) {
```

Comment:
Add `@Nonnull` annotation and  `final` modifier:
```suggestion
    public List<Bucket> findBuckets(@Nonnull final BucketsQuery query) {
```